mgmt_cli_test.py

```diff
@@ -880,8 +880,8 @@ def test_repair_multiple_keyspace_types(self):  # pylint: disable=invalid-name
         mgr_cluster = manager_tool.get_cluster(cluster_name=self.CLUSTER_NAME) \
             or manager_tool.add_cluster(name=self.CLUSTER_NAME, db_cluster=self.db_cluster,
                                         auth_token=self.monitors.mgmt_auth_token)
-        self._create_keyspace_and_basic_table(self.SIMPLESTRATEGY_KEYSPACE_NAME, "SimpleStrategy", replication_factor=2)
-        self._create_keyspace_and_basic_table(self.LOCALSTRATEGY_KEYSPACE_NAME, "LocalStrategy")
+        self._create_keyspace_and_basic_table(self.NETWORKSTRATEGY_KEYSPACE_NAME, replication_factor=2)
+        self._create_keyspace_and_basic_table(self.LOCALSTRATEGY_KEYSPACE_NAME, replication_factor=0)
```

Comment:
why changing it to `replication_factor=0` instead of `"LocalStrategy"`?
and why `0`?

Reply:
I wanted to simplify the code as strategy param seemed to be redundant. There's only one special case we want to use `Local` - which I found natural to a keyspace with RF=0 as `Local` is not replicated.